resources/zh/cemu.po

```diff
@@ -2021,88 +2594,281 @@ msgstr "DLC"
 msgid "&Save directory"
-msgstr "存档目录"
+msgstr "&存档目录"
 
 #: wxGameList.cpp
 msgid "&Update directory"
-msgstr "更新档目录"
+msgstr "&更新档目录"
 
 #: wxGameList.cpp
 msgid "&DLC directory"
-msgstr "DLC目录"
+msgstr "&DLC 目录"
 
 #: wxGameList.cpp
 msgid "&Edit graphic packs"
-msgstr "编辑图形包"
+msgstr "&编辑图形插件"
 
 #: wxGameList.cpp
 msgid "&Edit game profile"
-msgstr "编辑游戏设定"
+msgstr "&编辑游戏设定"
 
 #: wxGameList.cpp
 msgid "&Refresh game list"
-msgstr "刷新游戏列表"
+msgstr "&刷新游戏列表"
 
 #: wxGameList.cpp
 msgid "Style: &List"
-msgstr "显示方式:列表"
+msgstr "显示方式：&列表"
 
 #: wxGameList.cpp
 msgid "Style: &Icons"
-msgstr "显示方式:图标"
+msgstr "显示方式：&普通图标"
 
 #: wxGameList.cpp
 msgid "Style: &Small Icons"
-msgstr "显示方式:小图标"
+msgstr "显示方式：&小图标"
 
 #: wxGameList.cpp
 msgid "Reset &width"
-msgstr "重置宽度"
+msgstr "重置 &宽度"
 
 #: wxGameList.cpp
 msgid "Reset &order"
-msgstr "重置顺序"
+msgstr "重置 &排序"
 
 #: wxGameList.cpp
 msgid "Show &name"
-msgstr "显示名称"
+msgstr "显示 &名称"
 
 #: wxGameList.cpp
 msgid "Show &version"
-msgstr "显示版本"
+msgstr "显示 &版本"
 
 #: wxGameList.cpp
 msgid "Show &dlc"
-msgstr "显示DLC"
+msgstr "显示 &DLC"
 
 #: wxGameList.cpp
 msgid "Show &game time"
-msgstr "显示游戏时间"
+msgstr "显示 &游戏时长"
 
 #: wxGameList.cpp
 msgid "Show &last played"
-msgstr "显示上次游玩时间"
+msgstr "显示 &上次游玩时间"
+
+#: wxGameList.cpp
+msgid "Show &region"
+msgstr "显示 &地区"
+
+#: wxGameList.cpp
+msgid "You can configure game paths in the general settings."
+msgstr "可在通用设置中设定游戏路径。"
+
+#: wxTitleManagerList.cpp
+msgid "Title id"
+msgstr "档案 ID"
+
+#: wxTitleManagerList.cpp
+msgid ""
+"There's an wrong title installed at the target location, should we fix that?"
+msgstr "在目标位置中已经安装了一个错误的档案，我们要修复它吗？"
+
+#: wxTitleManagerList.cpp
+msgid ""
+"There's already a newer or equal version installed at the correct location, "
+"so the redundant version will be deleted:"
+msgstr "正确的目录中已经安装了一个更新的或相同的版本，故多余的版本将删除："
+
+#: wxTitleManagerList.cpp
+msgid ""
+"Error when trying to delete the entry:\n"
+"{}"
+msgstr ""
+"删除条目的过程中出错：\n"
+"{}"
+
+#: wxTitleManagerList.cpp
+msgid ""
+"There's already an older version installed at the correct location, so the "
+"older version will be overwritten:"
+msgstr "在正确的目录下已经安装了一个旧的版本，如果继续则旧的版本将被覆盖："
+
+#: wxTitleManagerList.cpp
+msgid ""
+"Error when trying to move the entry to the correct location:\n"
+"{}"
+msgstr ""
+"移动条目到正确位置时出错：\n"
+"{}"
+
+#: wxTitleManagerList.cpp
+msgid "This base game is installed at the wrong location."
+msgstr "这个游戏本体安装在了一个错误的位置上。"
+
+#: wxTitleManagerList.cpp
+msgid "This update is installed at the wrong location."
+msgstr "这个更新档安装在了一个错误的位置上。"
+
+#: wxTitleManagerList.cpp
+msgid "This DLC is installed at the wrong location."
+msgstr "这个 DLC 安装在了一个错误的位置上。"
+
+#: wxTitleManagerList.cpp
+msgid "You can use the context menu to fix it."
+msgstr "你可以使用右键菜单来修复它。"
+
+#: wxTitleManagerList.cpp
+msgid "&Launch title"
+msgstr "&载入档案"
+
+#: wxTitleManagerList.cpp
+msgid "&Open directory"
+msgstr "&打开目录"
+
+#: wxTitleManagerList.cpp
+msgid "Fix entry"
+msgstr "修复入口 | Fix entry"
```

Comment:
If you prefer, you can translate it as "Fix issues" instead of "Fix entry" (it fits better imho). Then remove the English version.

Reply:
Of course, I couldn't find a more suitable translation before, and I will change it in a later submission.